cmd/sportsball.go

```diff
@@ -11,7 +11,7 @@ Enjoy all your favourite Sportsball highlights, powered by
 https://github.com/cedricblondeau/world-cup-2018-cli-dashboard
 `)
 
-	task := command.Task("cedricbl/world-cup-2018-cli-dashboard")
+	task := command.Task("cdocker.io/edricbl/world-cup-2018-cli-dashboard")
```

Comment:
Looks like a typo here

Reply:
Well spotted

Reply:
I've just removed the command in d8c7c86. It's no longer useful

(not that it was ever particularly useful to begin with)